android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddm/form/DDMFormView.kt

```diff
@@ -280,7 +279,11 @@ class DDMFormView @JvmOverloads constructor(
 
 			setFieldVisibility(fieldContext, fieldView)
 			fieldTextView?.setupFieldLayout()
-			fieldViewModel?.refresh()
+
+			fieldViewModel?.let {
+				it.refresh()
+				it.setUpdateMode(!field.isReadOnly)
```

Comment:
Should it not be inside refresh method of Document Field? It's very strange that DDMFormView is the one responsible for enable or disable the document field

Reply:
Yeah, it's strange, but the problem is that the DDLFormView does the same, it's responsible to enable/disable DDL fields. I suggested @mclomello to keep the same on DDM to avoid side effects moving this logic to refresh in all field VMs (as he did initially).

Reply:
I don't know if we should keep this way, but we can take a look together in today's meeting.